public/dir.js

```diff
@@ -70,10 +72,16 @@
         const buttons = document.getElementsByClassName('delete-button');
         for (let button of buttons) {
           button.addEventListener('click', (e) => {
-            let deleteRequest = new XMLHttpRequest();
-            deleteRequest.onload = removeListItem(e.currentTarget);
-            deleteRequest.open('DELETE', `${window.location.pathname}/${e.currentTarget.parentElement.firstElementChild.innerText}`, true);
-            deleteRequest.send();
+            if (window.confirm(`Are you sure you want to delete ${e.currentTarget.parentElement.firstElementChild.innerText}`)) {
+              let deleteRequest = new XMLHttpRequest();
```

Comment:
Please prefer fetch over xhr, the latter is quite outdated.

https://developers.google.com/web/updates/2015/03/introduction-to-fetch

Reply:
Changing the XMLHttpRequest to Fetch is on the backlog for a while now. The thing is, the original code (the one before I started working on the project) also used only (and a lot of) XMLHttpRequests - the only Fetch we have is the one you made, to get the file names and create the list. 

Also, I thought you said code review would only happen later, after the prototype is out and running. Because of that, I didn't think it was a big deal now (you never said anything about the XMLHttpRequests on the original source). 

But I'll change it right now, no problem. I'd like to update the whole source though, not only this request (like I said, we have some XMLHttpRequests from the first version).

Reply:
fair.